src/frontend/src/components/common/InputText/InputText.tsx

```diff
@@ -71,7 +72,9 @@ const InputText = ({
   );
 
   useEffect(() => {
-    const inputRefCurrent = inputRef?.current as unknown as HTMLInputElement;
+    const inputRefCurrent = inputRef?.current as unknown as
+      | HTMLInputElement
+      | HTMLTextAreaElement;
```

Comment:
Look to line 64 and instantiate the inputRef like so: `const inputRef = useRef<HTMLInputElement | HTMLTextAreaElement>(null)`

Refer to: https://www.robinwieruch.de/typescript-react-useref/